### ISSUE TITLE
代码相关问题

### ISSUE THREAD

**Issue by Wanghc233** [HongyuYann/SymmCompletion#14]
模型文件第95行代码 feat = torch.cat([feat, gf_feat.repeat(1, feat.size(2), 1)], dim=-1) # B,640,512中的feat.size(2)是不是应该写成feat.size(1)

**Comment by hanryyan** [comment]
Right, thank you for your reminder. We have fixed this issue. By the way, because N and C are equal to 512 in our setting, it does not affect the final result.

**Comment by shiboliu** [comment]
feat: [B, 512, 512]
feat.size(1) 和 feat.size(2) 都可以

但是您的代码中注释可能存在问题，误导
